### ISSUE TITLE
mac screen .screen_layout, and layout not available

### ISSUE THREAD

**Issue by brian-lamb-software-engineer** [brian-lamb-software-engineer/bashetize#3]
If when you run screen, e.g. `srad t1`, t1 being a nickname for screen tab 1, and you see that it can not open ~/etc/.screen_layout, and `layout` command not found, you are probably using an incompatible, or older version of screen.  The following  will point you in the right direction .
Use brew to install screen. 

follow one of the methods below to link/alias the correct version of gnu screen

alias method
------------
open `~/etc/.bash-alias` file, and add the alias for it. e.g. 
```
#screen
alias screen="/usr/local/homebrew/Cellar/screen/4.9.0/bin/screen"
```

Symlink method.
-----------------

  Make sure your `/User/<username>/bin` folder is earlier on the path, or you need to edit ~/etc/.bash-profile-global line that has home path at teh end, and move home path to the beginning.  e.g. from `PATH=$PATH:$HOME/.local/bin:$HOME/bin` to `  PATH=$HOME/bin:$PATH:$HOME/.local/bin`the Remove the symlink from `~/screen`, and replace it with  `ln -s /usr/local/homebrew/Cellar/screen/4.9.0/bin/screen`
  
  TODO  
since default mac screen currently is showing 4.00.03, that version isn't loading the layout. 
  1 remove the el6/screen symlink for mac, since that one isnt working for mac, not sure why yet maybe find out. 
 2 include newer binary of screen in package subdir, e.g. mac/screen, and alias it 
  or 
2 update the docs for mac to brew install screen, then alias it.  
  3 test
 4 check why el6/screen file no longer works on mac as a valid updated replacement and add support for el6/7/8 distros. 
 
 Bottom line, if you get a newer versino of screen that works on your system, 4.6 or after, add the symlink, then your good, screen should be able to load the newer command `layout` no problem.
